src/api/ChatsAPI.ts

```diff
@@ -0,0 +1,65 @@
+import BaseAPI from './BaseAPI';
+import {Options} from '../utils/httptransport';
+
+export interface Message {
+  chat_id: string,
+  content: string,
+  file: string,
+  id: number,
+  is_read: boolean,
+  time: string,
+  type: string,
```

Comment:
можно ограничить количество строк, например так
```suggestion
  type: "type1" | "type2" | "type3",
```
и тогда если мы присвоим этому свойству неправильную строку, то TS выдаст предупреждение

Reply:
уточни пожалуйста зачем мне присваивать переменной type несколько типов, если мне нужен только один конкретный string